src/Raven.Client/Documents/Queries/QueryData.cs

```diff
@@ -0,0 +1,24 @@
+//-----------------------------------------------------------------------
+// <copyright file="QueryData.cs" company="Hibernating Rhinos LTD">
+//     Copyright (c) Hibernating Rhinos LTD. All rights reserved.
+// </copyright>
+//-----------------------------------------------------------------------
+
+namespace Raven.Client.Documents.Queries
+{
+    public class QueryData
```

Comment:
I wonder if `QueryData` name isn't too wide for this class. Maybe `QuerySelectData`, `QuerySelection`